src/EditorFeatures/CSharp/Formatting/CSharpEditorFormattingService.cs

```diff
@@ -39,7 +39,7 @@ public CSharpEditorFormattingService()
         public bool SupportsFormatDocument => true;
         public bool SupportsFormatOnPaste => true;
         public bool SupportsFormatSelection => true;
-        public bool SupportsFormatOnReturn => true;
+        public bool SupportsFormatOnReturn => false;
```

Comment:
why do we need this IEditorFormattingService stuff?

Reply:
so this is why format command handler suddenly start to handle "enter" since this service have this "SupportsFormatOnReturn" stuff. can we get rid of this one, why do we need this? format command handler should be able to handle all these?